01_git/scissors/scissors.c

```diff
@@ -0,0 +1,73 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+
+static char *options[] = {
+    "rock\0",
+    "paper\0",
+    "scissors\0"
+};
+
+static char *options_user[] = {
+    ['r'] = "rock\0",
+    ['p'] = "paper\0",
+    ['s'] = "scissors\0"
```

Comment:
Unused \0

Reply:
Fixed in https://github.com/Kernel-GL-HRK/gl_kernel_procamp_2021/pull/15/commits/b96e5a0237c2a6c2f7a8cc99f7905f573e7b7697